src/main/java/com/ibanity/apis/client/products/isabel_connect/models/AccountReport.java

```diff
@@ -0,0 +1,33 @@
+package com.ibanity.apis.client.products.isabel_connect.models;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.math.BigInteger;
+import java.time.Instant;
+import java.util.List;
+
+@Data
+@SuperBuilder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class AccountReport {
```

Comment:
Because this endpoint doesn't return some JSON: https://documentation.ibanity.com/isabel-connect/api#get-account-report
Hence the absence of an ID and the fact that it doesn't extend IsabelModel